### DISCUSSION TITLE
part-ii/2019-142/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#269] [Question comments]
Part II 2019 Statistical Physics: Paper 2, Section II, D 

https://questions.tripos.org/part-ii/2019-142/

**Comment by thenewtonium** [comment]
<details><summary>Point about first part</summary>

Note that to derive the ideal gas law, the only assumption we need to make is that the (overall) Hamiltonian is independent of the positions of the particles, since this will give us that $\log Z = \log (V^N) +$ [const. in $V$]; and hence $p = k_B T \frac{\partial}{\partial V} \log Z = N k_B T / V$.

</details>

<details><summary>Second part</summary>

I got that $pV^{5/2}$ is constant at constant $S$. There's not much to say other than this is an algebra bash, and that the entropy of a monatomic gas is a standard result from lectures (Sackur-Tetrode equation).

</details>
